src/renderer/components/hotbar/hotbar-menu.tsx

```diff
@@ -15,8 +22,19 @@ interface Props {
 @observer
 export class HotbarMenu extends React.Component<Props> {
 
+  componentDidMount() {
+    disposeOnUnmount(this, [
+      reaction(() => hotbarStore.activeHotbarId, () => {
+        Notifications.info(`Hotbar "${hotbarStore.getActive().name}" is now active.`, {
+          id: "active-hotbar",
+          timeout: 5_000
+        });
+      })
+    ]);
+  }
```

Comment:
I too think that this is a bit obnoxious. Would it be possible to do a sliding animation on the hotbar instead?

Reply:
Not sure how to do that without hacks. Maybe I'll just remove notifications from this PR?

Reply:
Sounds good probably a good idea to have it in a separate PR.

I would do it by rendering all of them but explicitly set the bounding box sizes and the `overflow-display: none` and then use JS powered css vars to change the animation placement. 

At least that is how I would start the impl attempt.